torchtext/csrc/register_pybindings.cpp

```diff
@@ -220,6 +220,32 @@ PYBIND11_MODULE(_torchtext, m) {
       .def(py::init<const std::string, bool, c10::optional<bool>>())
       .def("encode", &BERTEncoder::Encode)
       .def("tokenize", &BERTEncoder::Tokenize)
+      .def(
+          "batch_encode",
+          [](const c10::intrusive_ptr<BERTEncoder>& self,
```

Comment:
Is there a reason we have the operator implementation in the registration file as opposed to the `bert_tokenizer.cpp` file? I assume we'd like to keep registration code separate from the actual implementation code?

Reply:
That's a good catch. I moved the core implementation back into the class.